pgmpy/base/DAG.py

```diff
@@ -356,3 +356,36 @@ def get_children(self, node):
         ['D', 'E', 'F']
         """
         return list(self.successors(node))
+
+    def get_markov_blanket(self, node):
+        """
+        Returns a markov blanket for a random variable. In the case
+        of Bayesian Networks, the markov blanket is the set of
+        node's parents, its children and its children's other parents.
+
+        Returns
+        -------
+        list(blanket_nodes): List of nodes contained in Markov Blanket
+
+        Parameters
+        ----------
+        node: string, int or any hashable python object.
+              The node whose markov blanket would be returned.
+
+        Examples
+        --------
+        >>> from pgmpy.models import BayesianModel
```

Comment:
The documentation here needs to change, I think all instances of BayesianModel should be replaced by DAG.

Reply:
AAh yes you are right, missed that

Reply:
Fixed

Reply:
Looks good.